lib/rubycritic/commands/base.rb

```diff
@@ -9,7 +9,7 @@ def initialize(options)
       end
 
       def execute
-        fail NotImplementedError
+        raise NotImplementedError
```

Comment:
I do appreciate using `fail` to create an exception and `raise` to re-raise it and I believe Rubocup also suggests that.
But I often overlook these things and just type `raise` everywhere.

That being said, and since we're trying to help people write better ruby code, we should follow the community trends.
I vote to stick with `fail` here.